src/Typescript/Server.ts

```diff
@@ -28,30 +28,34 @@ interface TurnInfo {
 
 export function turnTaken(turnInfo: TurnInfo) {
     var turn = document.createElement("div");
-    turn.textContent = "Player " + players[currentPlayer].position + " "
+    turn.className += "log-row";
+    turn.textContent = players[currentPlayer].position + " "
             + (turnInfo.clueType
-                    ? "Clued player " + turnInfo.player.position + " on "
-                    : turnInfo.discard ? "Discarded" : "Played") + " ";
+                    ? "clued " + turnInfo.player.position + " on "
+                    : turnInfo.discard ? "discarded" : "played") + " ";
     if (turnInfo.tile) {
         var clueOnNumber = typeof(turnInfo.clueType) === "number";
         var clueOnColor = typeof(turnInfo.clueType) === "string";
-        var textContent = String(turnInfo.tile.number) + (turnInfo.clueType ? "s" : "");
+        var textContent = String(turnInfo.tile.number);
         if (clueOnColor) {
             textContent = turnInfo.tile.color;
         }
         if (turnInfo.clueType) {
             textContent +=
                     " ("
                     + turnInfo.indices.map(i => ["first", "second", "third", "fourth"][i]).join(", ")
-                    + ")";
+                    + " tile)";
         }
         var color = Util.colorMap[turnInfo.tile.color];
+        var tileDescription = document.createElement("span");
         if (clueOnNumber) {
             color = "";
+            tileDescription.className = "inline-tile-blank";
+        } else {
+            tileDescription.className = "inline-tile" + (color == "#EAEDD5" ? " font-color-black" : "");
```

Comment:
I'd prefer either `Util.colorMap["white"]` or `turnInfo.tile.color === "white"` to the hardcoded hex value.